oscarapi/tests/unit/testbasket.py

```diff
@@ -1061,12 +1061,22 @@ def test_add_product_above_stock(self):
         """Test if an anonymous user cannot add more products to his
             basket when stock is not sufficient
         """
-        self.response = self.post(
-            "api-basket-add-product",
-            url="http://testserver/api/products/1/",
-            quantity=25,
-        )
-        self.response.assertStatusEqual(406)
+        with self.subTest("Single request"):
+            self.response = self.post(
+                "api-basket-add-product",
+                url="http://testserver/api/products/1/",
+                quantity=25,
+            )
+            self.response.assertStatusEqual(406)
+
+        with self.subTest("Sequential requests"):
```

Comment:
It would be nice to choose an amount here that will be allowed the first time, but will be rejected the second time because the double that amount will trigger the error. so if 10 is allowed add 10 twice, the first will good, the second will fail.

Reply:
It does that in a loop below, but I'll add extra asserts so it's more obvious.

Reply:
Done.